test/docker-compose.yml

```diff
@@ -1,6 +1,12 @@
-# Configure the services needed for the integration tests.
-# More information at https://docs.docker.com/compose/
 version: "3.9"
 services:
-  hello-world:
-    image: "hello-world:latest"
+  cassandra:
+    image: cassandra:latest
```

Comment:
I see pros in testing the latest image always, but it can also create surprises when the image changes.